modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -115,13 +115,74 @@ authService.login().then(() => {
 IAuthService authService = MobileCore.Instance.GetService<IAuthService>();
 
 var authOptions = DependencyService.Get<IAuthenticateOptionsProvider>().GetOptions();
-authService.Authenticate(authOptions).ContinueWith(user =>
-{
-    // Login success
+var user = await authService.Authenticate(authOptions);
+----
+****
+
+== Refreshing the authentication token
+
+When a user is authenticated, an access token and a refresh are generated by the authentication server.
+See link:https://www.keycloak.org/docs/3.2/server_admin/topics/sessions/timeouts.html[here] for configuring the lifespan of these tokens.
+If the refresh token is still valid, the authentication token can be refreshed even if the authentication token has expired.
+This operation is supported and can be automatically done with the Aerogear SDK as shown below.
```

Comment:
trying to avoid 'this operation is supported', cos I don't know what that means? suggests other things aren't supported, we leave this type of statement to supported configs pages, every published procedure should represent what we want to support, right?